src/notation/internal/notationparts.cpp

```diff
@@ -1263,26 +1320,33 @@ void NotationParts::updateSoloist(const PartInstrumentList& parts)
     }
 }
 
-void NotationParts::sortParts(const PartInstrumentList& parts)
+void NotationParts::sortParts(const PartInstrumentList& parts, const QList<Ms::Staff*>& originalStaves)
 {
     Q_ASSERT(score()->parts().size() == static_cast<int>(parts.size()));
 
-    for (int i = 0; i < parts.size(); ++i) {
-        const Part* currentPart = score()->parts().at(i);
+    QList<int> staffMapping;
+    QList<int> trackMapping;
+    int idx2 = 0;
```

Comment:
Probably we can rename this variable. The current name doesn't look too obviously

Reply:
It isn't obvious at all but just copy of original 3.x code where apparently people suffered from find a decent name :smiley: 
I'll give it another try,

Reply:
Done.